src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormDeviceSelect.tsx

```diff
@@ -0,0 +1,37 @@
+import { type Control, Controller, type FieldValues } from 'react-hook-form'
+
+import { ContentHeader } from '../layout/ContentHeader.js'
+import { ThermostatSelect } from '../thermostat/ThermostatSelect.js'
+import { t } from './ClimateSettingScheduleForm.js'
+
+interface ClimateSettingScheduleFormDeviceSelectProps {
+  control: Control<FieldValues, any>
+  onSelect: () => void
+  onBack: (() => void) | undefined
+}
+
+export function ClimateSettingScheduleFormDeviceSelect({
+  control,
+  onBack,
+  onSelect,
+}: ClimateSettingScheduleFormDeviceSelectProps): JSX.Element {
+  return (
+    <>
+      <ContentHeader title={t.addNewClimateSettingSchedule} onBack={onBack} />
```

Comment:
title could be a prop passed in from the parent

Reply:
[e1c6eab](https://github.com/seamapi/react/pull/460/commits/e1c6eab7e4bc2547e8364137a8eee34e71a4408e)